app/views/pages/get-started/build-your-first-app/removing-data-from-the-database.liquid

```diff
@@ -46,7 +46,7 @@ If you are following this tutorial from the start, you already know what to do.
 Run the following command to start the administration interface of your platformOS instance:
 
 ```bash
-pos-cli gui serve <instance alias>
```

Comment:
Those were suppose to be a universal instructions as much as possible, that is the reason for not enforcing a previously mentioned instance alias.

Reply:
Agreed with @nemvalid regarding being able to copy-paste the command and expect it to work. I also like the point about adding universal instructions, I am wondering if we can have both if we make it clear where the `dev` is coming from? One of the idea how to do it could be to phrase is something like:
`Run the pos-cli gui serve <instance alias> command to start the administration interface of your platformOS instance, in our scenario we used "dev" as an instance alias, so the command would be:`